OrbitVulkanLayerClientGgp/CMakeLists.txt

```diff
@@ -2,21 +2,33 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-project(OrbitVulkanLayerClientGgp)
-add_library(OrbitVulkanLayerClientGgp SHARED)
+project(OrbitCaptureVulkanLayer)
```

Comment:
I really don't want to overemphasis this naming stuff - fell free to ignore... OrbitTriggerCaptureVulkanLayer would be way more destinctive  imo.

Reply:
That was a quick change, so I have renamed it. I agree it makes sense. Thanks for the suggestion!